docs/changelog.rst

```diff
@@ -15,6 +15,62 @@ IN DEVELOPMENT
 
 Changes will be summarized here periodically.
 
+Version 2.10.2 (Dec 20, 2022)
+-----------------------------
+
+Changes:
+
+* ``scoped_interpreter`` constructor taking ``PyConfig``.
+  `#4330 <https://github.com/pybind/pybind11/pull/4330>`_
+
+* ``pybind11/eigen/tensor.h`` adds converters to and from ``Eigen::Tensor`` and
+  ``Eigen::TensorMap``.
+  `#4201 <https://github.com/pybind/pybind11/pull/4201>`_
+
+* Add option for enable/disable enum members in docstring.
+  `#2768 <https://github.com/pybind/pybind11/pull/2768>`_
+
+Bug fixes:
+
+
```

Comment:
Is the 2nd empty line intentional?

Reply:
No, though it's a bit random currently on older entries too. When we move to markdown I'll try to use prettier.